conf/test.config

```diff
@@ -27,18 +28,18 @@ process {
     errorStrategy = params.err_start
 
     withLabel:process_low {
-        cpus   = { check_max( 1                  , 'cpus'    ) }
-        memory = { check_max( 4.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 10.m  * task.attempt, 'time'    ) }
+        cpus   = { 2                    }
```

Comment:
this changed from 1 to 2, was it on purpose?

Reply:
Good spot!

Reply:
I actually, now remember to set a similar default to other pipelines but TBH, don't have a strong opinion, will revert